app/exceptions/handlers.py

```
@@ -0,0 +1,85 @@
+
+import app.exceptions.exception_constants
+from app.exceptions.custom_exceptions import DevDoxAPIException
+from app.utils.api_response import APIResponse
+
+logger = logging.getLogger(__name__)
+
+generic_exception_handler_status_code = status.HTTP_503_SERVICE_UNAVAILABLE
+
+
+def generic_exception_handler(request: Request, exc: Exception) -> dict[str, Any]:
+	path = request.url.path
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)